egs2/oniku/svs1/local/prep_segments.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
```

Comment:
can we have possibilities to use the same one as ofuton as they are very similar?

Reply:
They're the same.